app/controllers/application_controller.rb

```diff
@@ -4,6 +4,7 @@ class ApplicationController < ActionController::Base
   include SessionCurrentUser
   include ValidRequest
   include Pundit
+  include AuditInstrumentation
```

Comment:
I'm not super convinced this needs to be a concern. All the data it needs gets passed in. I'd be happier with `notify` just being a class method on some module/class, e.g. `Audit.notify` or something.

Reply:
+1 on this, I think auditing should be as explicit as possible.

I also think the word `notify` clashes a bit as a name for the act of auditing or logging user operations. What about `Audit.log` ? But yeah, less sure about this

Reply:
Update this a little bit, let me know what you think @rhymes @citizen428!